src/Compilers/CSharp/Portable/Binder/LocalBinderFactory.cs

```diff
@@ -305,7 +305,17 @@ public override void VisitEqualsValueClause(EqualsValueClauseSyntax node)
 
         public override void VisitAttribute(AttributeSyntax node)
         {
-            var attrBinder = new ExpressionVariableBinder(node, _enclosing);
+            Binder attrBinder = null;
+            if (node.Parent?.Parent is { } attributedDeclaration)
+            {
+                var model = _containingMemberOrLambda.DeclaringCompilation.GetSemanticModel(node.SyntaxTree);
+                if (model.GetDeclaredSymbolForNode(attributedDeclaration) is { } attributedSymbol)
```

Comment:
This call causes a cycle. Any suggestion on how to get the attributed member to properly create `ContextualAttributeBinder`?